src/data/file.ts

```diff
@@ -49,6 +49,18 @@ export interface FileDetails {
    * license will take priority over the global license.
    */
   license?: string;
+
+  /**
+   * A reference to the document that was used to create this File class. Used as an optimization
+   * where applicable. If passed, 'pageIndex' must also be passed
+   */
+  fullDocumentObj?: CoreControls.Document
+
+  /**
+   * Used in conjunction with 'fullDocumentObj'. Represents the pageIndex of 'fullDocumentObj' that this
+   * file belongs too
+   */
+  pageIndex?: number
```

Comment:
(Just removing spaces and adding semicolons

```suggestion
  license?: string;
  /**
   * A reference to the document that was used to create this File class. Used as an optimization
   * where applicable. If passed, 'pageIndex' must also be passed
   */
  fullDocumentObj?: CoreControls.Document;
  /**
   * Used in conjunction with 'fullDocumentObj'. Represents the pageIndex of 'fullDocumentObj' that this
   * file belongs too
   */
  pageIndex?: number;
```